services/QuillLMS/app/views/navbar/_teacher_center.html.erb

```diff
@@ -10,6 +10,16 @@
       link: '/teacher-center',
       text: 'All Resources'
     %>
+    <% if current_user && AppSetting.enabled?(name: 'comprehension', user: current_user) %>
```

Comment:
It probably makes sense to make `'comprehension'` a const in the `AppSetting` model file.